modules/cij/runner.py

```diff
@@ -789,7 +789,11 @@ def trun_setup(args: argparse.Namespace, conf: cij.conf.Config) -> TestRun:
     trun.res_root = args.output
     trun.aux_root = os.path.join(trun.res_root, "_aux")
 
-    os.makedirs(trun.aux_root)
+    try:
+        os.makedirs(trun.aux_root)
+    except CIJError as ex:
```

Comment:
Ah, now I get what you mean.
I don't think ``os.makedirs()`` would ever throw ``CIJError``?

How about catching ``FileExistsError`` here, then wrapping it in a new cijoe-exception e.g. ``InvalidResultAuxPath`` by adding it to ``modules/cij/errors.py`` and raising it with something like:
```
except FileExistsError as exc:
   cij.err("trun_setup: FAILED: os.makedirs(). Error: %r" % exc)
   raise InvalidResultAuxPath("foo") from exc
```

Reply:
I have updated the PR.